src/fileio.cpp

```diff
@@ -16,16 +16,15 @@
 #include "fileio.h"
 #include "rev.h"
 #ifdef LINUX
-	#include "unix/fileio_unix.h"
 	#include <dirent.h>
 	#include <errno.h>
 	#include <unistd.h>
 #elif WINDOWS
-	#include "windows/fileio_windows.h"
 	#include <direct.h> // contains chdir in windows
 #endif
 #include <sys/types.h>
-#include <sys/stat.h>
+
+const char *DIR_SEP = "/";  ///< Directory separator character.
```

Comment:
How come you're not using `std::filesystem::path::preferred_separator`?

Reply:
You're right, done :)